stylefunction.js

```diff
@@ -601,7 +602,18 @@ export default function(olLayer, glStyle, source, resolutions, spriteData, sprit
             label = label.toLowerCase();
           }
           const wrappedLabel = type == 2 ? label : wrapText(label, font, getValue(layer, 'layout', 'text-max-width', zoom, f));
-          text.setText(wrappedLabel);
+          const letterSpacing = getValue(layer, 'layout', 'text-letter-spacing', zoom, f);
+          if (letterSpacing >= 0.05) {
+            let wrappedLabelWithLetterSpacing = '';
+            const wrappedLabelLines = wrappedLabel.split('\n');
+            const joinSpaceString = hairSpacePool.slice(0, Math.round(letterSpacing / 0.1));
+            for (let lineIndex = 0; lineIndex < wrappedLabelLines.length; lineIndex += 1) {
+              wrappedLabelWithLetterSpacing += wrappedLabelLines[lineIndex].split('').join(joinSpaceString);
```

Comment:
```suggestion
              wrappedLabelWithLetterSpacing += wrappedLabelLines[l].split('').join(joinSpaceString);
```

Reply:
Oh, and one more thing: you need to re-add the line breaks to wrappedLabelWithLetterSpacing.

Reply:
oops, yep forgot that